src/drivers/Bma421.cpp

```diff
@@ -68,11 +75,21 @@ void Bma421::Init() {
   if (ret != BMA4_OK)
     return;
 
+  // Configure FIFO
+  ret = bma4_set_fifo_config(BMA4_FIFO_ACCEL, 1, &bma);
+  if (ret != BMA4_OK)
+    return;
+  fifo_frame.data = (uint8_t*) &fifo;
+  fifo_frame.length = sizeof(fifo);
+  fifo_frame.fifo_data_enable = BMA4_FIFO_A_ENABLE;
+  fifo_frame.fifo_header_enable = 0;
+
+  // Configure accelerometer
   struct bma4_accel_config accel_conf;
-  accel_conf.odr = BMA4_OUTPUT_DATA_RATE_100HZ;
+  accel_conf.odr = BMA4_OUTPUT_DATA_RATE_200HZ;
   accel_conf.range = BMA4_ACCEL_RANGE_2G;
   accel_conf.bandwidth = BMA4_ACCEL_NORMAL_AVG4;
-  accel_conf.perf_mode = BMA4_CIC_AVG_MODE;
+  accel_conf.perf_mode = BMA4_CONTINUOUS_MODE;
```

Comment:
Do these changes have an impact on power usage? Do we need such frequent data?

Reply:
200 Hz is the maximum data rate which can be read by the watch, since the FIFO is able to capture 32 samples at most, and the sensor cycle time is 100 ms. This gives 20 samples per cycle at a sample rate of 200 Hz. I chose this sample rate because I needed the best possible data for #1133 .

`BMA4_CIC_AVG_MODE` and `BMA4_CONTINUOUS_MODE` should make no difference in power consumption, since the internal sample rate stays the same, the AVG mode just averages the samples together. I theory, operating the sensor in continuous mode even offloads some calculations from the sensor chip.

According to the datasheet at https://datasheet.lcsc.com/lcsc/1912111437_Bosch-Sensortec-BMA425_C437656.pdf , the power consumption in normal mode is not correlated to the sampling frequency. This is only the case in low power mode, which was not used here before and still is not used.